src/render/renderer-ecs.ts

```diff
@@ -423,7 +424,9 @@ async function chooseAndInitRenderer(
     // try webgpu first
     const adapter = await navigator.gpu?.requestAdapter();
     if (adapter) {
-      const device = await adapter.requestDevice();
+      const device = await adapter.requestDevice({
+        requiredFeatures: ["timestamp-query"],
```

Comment:
future todo: we should make this an optional feature request b/c there's a good chance our end users will never have this feature since precise timestamps are a potential security hazard.

Reply:
done